hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc

```diff
@@ -218,5 +219,39 @@ available and are observable as MBeans under the node
 `com.hazelcast/Metrics/<instanceName>/`.
 
 For further information on the IMDG metrics pleaso consult the
-{hz-refman}[Hazelcast IMDG Reference Manual]
-.
\ No newline at end of file
+{hz-refman}[Hazelcast IMDG Reference Manual].
+
+=== Job Metrics
+Since Jet 3.2 there is an alternative to JMX for monitoring metrics, albeit only
+the job-specific ones. `Job` objects have a `getMetrics()` method, which
+returns `JobMetrics` instances and those contain the latest known metric values
+for their source jobs.
+
+This functionality has been developed primarily for giving access to metrics of
+finished jobs, but can in fact be used for jobs with any {jet-javadoc}/core/JobStatus.html[status].
+
+While the job is running the metric values are updated periodically (see
+{jet-javadoc}/config/MetricsConfig.html#setCollectionIntervalSeconds-int-[MetricsConfig.setCollectionIntervalSeconds]).
+
+Once a job stops executing (successfully, after a failure, cancellation,
+or temporarily while suspended) the metrics will have their most
+recent values (i.e. the last metric values from the moment before the
+job completed).
+
+When a job is restarted then the metrics are reset too, their values
```

Comment:
We should mention here that metrics are also reset after calling `resume` (after previous `suspend`).